WordPress/WordPressUITests/Tests/SupportScreenTests.swift

```diff
@@ -0,0 +1,27 @@
+import XCTest
+
+class SupportScreenTests: XCTestCase {
+
+    override func setUp() {
+        setUpTestSuite()
+
+        LoginFlow.logoutIfNeeded()
+    }
+
+    override func tearDown() {
+        takeScreenshotOfFailedTest()
+        LoginFlow.logoutIfNeeded()
+        super.tearDown()
+    }
+
+    // Test Support Section Loads
+    // From Prologue > continue, tap "help" and make sure Support Screen loads
+    func testSupportScreenLoads() {
+        let supportScreen = PrologueScreen().selectContinue().selectHelp()
```

Comment:
A key advantage of the screen object pattern is that we can hide all the implementation details  behind the public API of the screen object. In this case, we could move the "continue and tap help" into the screen object and simplify this into:

```suggestion
        let supportScreen = PrologueScreen().loadSupportScreen()
```

Reply:
I think I disagree in this particular case it makes it less readable to me. If I'm following along on my device, `loadSupportScreen()`doesn't tell me anything. We have to tap "continue" before we have access to the "help" button, so it seems better as two separate steps. That way, reading the test makes clear what steps are being followed. 
Happy to discuss more when we migrate though!

Reply:
I'm glad you disagree. Thank you 😄 

> That way, reading the test makes clear what steps are being followed.

That's a totally legit point of view. It occurred to me, too, while making my comment. I guess I'm really excited about Screen Objects right now and so I'm leaning towards encapsulating as much as possible into them. I can't wait to have "in the trenches" experience with the pattern in the context of this and our other apps, so we can see whether "as much encapsulation as possible" is a good rule of thumb, or there is space for this syntax that is more sequential and exposes the high level action a user would have to take in the app 👀